nekoyume/Assets/_Scripts/UI/IconAndButton.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Nekoyume.UI.Module;
+using TMPro;
+using UnityEngine;
+
+namespace Nekoyume.UI
+{
+    public class IconAndButton : MonoBehaviour
```

Comment:
What is the intention of this class? It's name is `IconAndButton` but it doesn't have any images are something can handle icon.